src/cli_common.c

```diff
@@ -412,16 +414,15 @@ sds escapeJsonString(sds s, const char *p, size_t len) {
     return sdscatlen(s,"\"",1);
 }
 
-sds redisVersion(void) {
-    sds version;
-    version = sdscatprintf(sdsempty(), "%s", REDIS_VERSION);
-
-    /* Add git commit and working tree status when available. */
-    if (strtoll(redisGitSHA1(),NULL,16)) {
-        version = sdscatprintf(version, " (git:%s", redisGitSHA1());
-        if (strtoll(redisGitDirty(),NULL,10))
-            version = sdscatprintf(version, "-dirty");
-        version = sdscat(version, ")");
-    }
+sds getVersion(const char *title) {
+    sds version = sdscatprintf(sdsempty(),
+        "%s v=%s sha=%s:%d malloc=%s bits=%d build=%llx",
+        title,
+        REDIS_VERSION,
+        redisGitSHA1(),
+        atoi(redisGitDirty()) > 0,
+        ZMALLOC_LIB,
+        sizeof(long) == 4 ? 32 : 64,
+        (unsigned long long) redisBuildId());
```

Comment:
i'm not sure why you did this change.

i don't think we have to make the CLI version print exactly like the server's version print.
e.g. the zmalloc and bits aren't really needed.

maybe the reason you did that is for the redis-*-check one to be similar to the the benchmark and cli, i think it's acceptable that they'll have a version string that's similar to the server and different than the cli.

Reply:
my first idea is to make them consistent, but you are right, cli does not need these fields.

Reply:
i think we still better revert it to the same format it was before.
i don't mind about redis-check-rdb, but for redis-cli, i prefer to keep the same format it had, maybe someone wrote some script that parses it. :shrug:

Reply:
Sorry for spending your energy on such a small PR. I updated the PR now it will look like this:

old:
```
./src/redis-server --version
Redis server v=255.255.255 sha=5dc631d8:0 malloc=libc bits=64 build=9e41a22b276f9625

./src/redis-cli --version
redis-cli 255.255.255 (git:5dc631d8)

./src/redis-benchmark --version
redis-benchmark 255.255.255 (git:5dc631d8)

./src/redis-check-rdb --version
redis-check-rdb 255.255.255 (git:5dc631d8)
```

now:
```
 ./src/redis-server --version
Redis server v=255.255.255 sha=6fa65d2f:0 malloc=libc bits=64 build=ff7faa53a7a67b58

./src/redis-cli --version
redis-cli 255.255.255 (git:6fa65d2f)

./src/redis-benchmark --version
redis-benchmark 255.255.255 (git:6fa65d2f)

./src/redis-check-rdb --version
redis-check-rdb v=255.255.255 sha=6fa65d2f:0 malloc=libc bits=64 build=ff7faa53a7a67b58

./src/redis-check-aof --version
redis-check-aof v=255.255.255 sha=6fa65d2f:0 malloc=libc bits=64 build=ff7faa53a7a67b58
```